README.md

```diff
@@ -218,7 +218,8 @@ the specified context and less than the context for the next release.
 
 | jwst tag            | DMS build | SDP_VER  | CRDS_CONTEXT | Released   | Ops Install | Notes                                         |
 |---------------------|-----------|----------|--------------|------------|-------------|-----------------------------------------------|
-| 1.13.3              | B10.1rc4  | 2023.4.0 | 1181         | 2024-01-05 |             | Fourth release candidate for B10.1             |
+| 1.13.4              |           |          | 1185         | 2024-01-25 |             | PyPI-only release for external users          |
+| 1.13.3              | B10.1rc4  | 2023.4.0 | 1181         | 2024-01-05 |             | Fourth release candidate for B10.1            |
```

Comment:
While you're in here making mods you could update "Fourth release candidate" to "Final release candidate"